### ISSUE TITLE
Question about the raw data

### ISSUE THREAD

**Issue by lpmi-13** [RevanthRameshkumar/CRD3#2]
First off, awesome work with this project!

Just curious as to whether there are any plans to include the scripts used to download the raw data.

I noticed it was all in html form that contains a lot of ad/tracking cookies/scripts, and was just thinking it would be possible to preserve the fidelity of the data source while at the same time allowing end users the option to strip these out if they didnt need them.

Not immediately sure if they are needed, but I'm assuming they dont contribute any part to the cleaned data in json format, so thats why I'm asking.

**Comment by RevanthRameshkumar** [comment]
Yes absolutely! I actually tried to do all the extraction purely through code so that is reproducible.
Since you asked, I'll prioritize releasing the extractions scripts first 😄

**Comment by lpmi-13** [comment]
Fantastic! I'd be very interested in helping out with that if there's anything I can contribute.

I'm mostly experienced with python and javascript, but just lemme know if there's anything that would be useful, and I'm very willing to give it a shot.

Even just dockerizing the setup so script dependencies aren't so much of an issue is definitely something I can help out with if that would be beneficial.

**Comment by RevanthRameshkumar** [comment]
Hey! My coauthor and I are discussing next steps for scripts. Perhaps I can make open issues for people who are interesting in contributing to tackle.

**Comment by lpmi-13** [comment]
that sounds great!

**Comment by RevanthRameshkumar** [comment]
Hi @lpmi-13, sorry to be silent for a little bit, work has been busy :)
Still interested in contributing? I found a first task that might be interesting!

**Comment by lpmi-13** [comment]
Oh no worries at all. It's a weird time for everyone :)

Yes, definitely still interested! Go ahead and raise an issue and tag me,
or whatever you think best.

**Comment by RevanthRameshkumar** [comment]
Here you go @lpmi-13!
https://github.com/RevanthRameshkumar/CRD3/issues/5#issue-683241413
